### ISSUE TITLE
Option to spoof window.navigator.webdriver?

### ISSUE THREAD

**Issue by tim-stephenson** [openwpm/OpenWPM#1031]
[window.navigator.webdriver](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/webdriver) exposes if a client is "controlled by automation". Potentially sites could treat a crawler differently based on the if the `window.navigator.webdriver` is true, leading to a crawl less representative of a real user visiting a site. The browser parameter of `bot_mitigation` already exists, perhaps enabling `bot_mitigation` should also spoof  `window.navigator.webdriver` to be false?

In short, should this be spoofed to be false (or have there be an option to do so?)

**Comment by Giblin91** [comment]
There is this pull request on the matter: [#526](https://github.com/openwpm/OpenWPM/pull/526)
